ldi-orchestrator/ldio-connectors/ldio-rdf-adapter/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioRdfAdapterAutoConfig.java

```diff
@@ -14,7 +14,7 @@ public class LdioRdfAdapterAutoConfig {
 	public static final String MAX_JSONLD_CACHE_CAPACITY = "max-jsonld-cache-capacity";
 
 	@SuppressWarnings("java:S6830")
-	@Bean("be.vlaanderen.informatievlaanderen.ldes.ldi.RdfAdapter")
+	@Bean("Ldio:RdfAdapter")
 	public LdioAdapterConfigurator ldiHttpOutConfigurator() {
```

Comment:
This is more outside of the scope of this story, but I would expect this method name to be more like `ldioAdapterConfigurator` or `ldioRdfConfigurator`